andela/travel_tool_front
# #161445627 Add depature time to ticket details on checklist submission

#### What does this PR do?
- Adds the departure time field to the ticket form that will be filled during checklist submission
- Fixes CSS issues that arise from the addition of the above field

#### Description of Task to be completed?
- Add the departure time field
- Fix CSS issues that arise from the addition of the above field

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ft-add-departure-time-to-ticket-details-161445627** branch
   `git checkout ft-add-departure-time-to-ticket-details-161445627`
 - Install dependencies and start the server
   - `make start` *if you have docker*
   - `yarn install` then `yarn start` *if you do not have docker*
 - Start the [backend](https://github.com/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
- On your database, modify your **roleId** in the **UserRoles** table to **53019** (Manager).
- Add a new request.
- Approve the request.
- Click on the 3-dot menu then click on **Travel Checklist**
- You should see a modal with a linear form for adding the flight ticket details. Fill this form.
- Open your local database and view the data stored. Matching screenshots are shown below.

#### Any background context you want to provide?
For any bugs that are found on this PR and are also available on **develop**, a [story](https://www.pivotaltracker.com/story/show/162008362) has been created to deal with them which is beyond the scope of this PR.

#### What are the relevant pivotal tracker stories?
* [#161445627](https://www.pivotaltracker.com/story/show/162008362)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/41262970/48713680-044d0600-ec22-11e8-9d84-2d4b73658fbb.png)
![image](https://user-images.githubusercontent.com/41262970/48713689-07e08d00-ec22-11e8-94d1-97f36f2d060d.png)

#### Questions:
None
